lib/coqProject_file.ml

```diff
@@ -116,12 +116,11 @@ and parse_args buf accu s = match Stream.next s with
 | exception Stream.Failure -> accu
 
 let parse f =
-  let c = open_in f in
+  let c = try open_in f with Sys_error msg -> Format.eprintf "Error: @[%s@].@\n%!" msg; exit 1 in
```

Comment:
Do we really want this to be unrecoverable? Seems like it should just raise a Parsing_error

Reply:
You mean returning a parsing error if the project file to open cannot be opened? This looks confusing to me.

Reply:
Better than exit imo

Reply:
Perhaps we need an `UnableToOpenProjectFile` exception that reraises this exception. That way it is clear that the project file was unable to be opened and the reason why. I agree that exiting is probably not the best thing to do.

Reply:
I don't see what's confusing about `Parsing_error "Could not open project file foo"` tbh
alternatively we don't need to catch the sys_error

Reply:
As `coqdep` is currently done, it reports uncaught errors as anomalies, so we need to do something. I'd be tempted to adopt the `UnableToOpenProjectFile` proposal.